app-accessibility/orca/orca-43.1.ebuild

```diff
@@ -0,0 +1,62 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+PYTHON_COMPAT=( python3_{8..11} )
+
+inherit gnome2 python-single-r1
+
+DESCRIPTION="Extensible screen reader that provides access to the desktop"
+HOMEPAGE="https://wiki.gnome.org/Projects/Orca"
+
+LICENSE="LGPL-2.1+ CC-BY-SA-3.0"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~ia64 ~loong ~ppc ~ppc64 ~riscv ~sparc ~x86"
+
+IUSE="+braille"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+DEPEND="${PYTHON_DEPS}
+	>=app-accessibility/at-spi2-atk-2.34:2
+	>=app-accessibility/at-spi2-core-2.36:2[introspection]
+	>=dev-libs/atk-2.36
```

Comment:
at-spi2-atk and atk are not valid anymore. You have only to leave at-spi2-core as it superseed other 2

Reply:
Done in https://github.com/gentoo/gentoo/commit/4739486fa69f15955c8b9ece7984a01a6c04f3df